src/Symfony/Component/Console/Event/ConsoleEvent.php

```diff
@@ -38,7 +38,7 @@ public function __construct(Command $command, InputInterface $input, OutputInter
     /**
      * Gets the command that is executed.
      *
-     * @return Command A Command instance
+     * @return Command|null A Command instance
```

Comment:
Yeah see https://github.com/symfony/symfony/pull/18140#r55926614 :

 > Strictly speaking, this is a BC break. But only the public API is important for event classes, so I don't think we should consider this a BC break.